scripts/generate_db.bat

```diff
@@ -0,0 +1,8 @@
+pushd condensation-forum
```

Comment:
Prepend `@echo off` to disabling the current directory printing on every command executed. Append `@echo on` to re-enable this at the end. It's a minor detail with Batch, but can make the output cleaner.